src/OrleansRuntime/Catalog/Catalog.cs

```diff
@@ -772,6 +772,13 @@ internal void ShutdownActivation_DeactivateOnIdle(ActivationData data)
             }
         }
 
+        public Task ShutdownActivations_DeactivateAll()
```

Comment:
This is against the general naming convention. DeactivateAllActivation() is probably a better name.